hack/README.md

```diff
@@ -0,0 +1,89 @@
+# Hack
+
+This directory includes convenience scripts for tools that assist developers in creating local Kubernetes clusters (using Docker) and then deploying/configuring Tekton components into them.
+
+## Script overview
+
+| Script | Description |
+| :-- | :-- |
+| [tekton_in_kind.sh](#tekton_in_kindsh) | Stands up a K8s cluster using the [kind](https://kind.sigs.k8s.io/) tool and deploys Tekton `pipeline`, `triggers` and `dashboard` components. |
+| [tekton_ci.sh](#tekton_cish) | Sets up a GitHub webhook to a fork of the `tekton/plumbing` repo. using using the `smee` tool. |
```

Comment:
one more instance of `using using` here but otherwise this looks great!

Reply:
fixed